### DISCUSSION TITLE
Add support for GPT-4 Vision

### DISCUSSION THREAD

**Discussion by golfista** [ChatGPTNextWeb/NextChat#3144] [Ideas]
It is currently available for everyone (i believe) with GPT-4 access, for the image hosting we can use S3 amazon buckets, google GCP and/or bunny.net storage

**Comment by qwe325207** [comment]
Add dall-e-3 to the custom model. When using it, it shows that I don’t have permission.

{
  "error": {
    "message": "You are not allowed to sample from this model",
    "type": "invalid_request_error",
    "param": null,
    "code": null
  }
}

**Comment by jacobjiangmj** [comment]
we need this. my author dadaa plz, support it.

**Comment by H0llyW00dzZ** [comment]
already added
